pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.poetry]
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "^3.11.0"
+promptflow = "^1.6.0"
+promptflow-tools = "^1.3.0"
+ipython = "^8.22.2"
+ipykernel = "^6.29.3"
+nbstripout = "^0.7.1"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^8.1.1"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+line-length = 240
```

Comment:
We all use wide monitors, and unless the otherwise is convinced, I think we go bigger 😏